generator/main.go

```diff
@@ -124,4 +130,5 @@ func main() {
 				n.Oid, n.Label, t, n.TextualConvention, n.Hint, n.Indexes, implied, n.EnumValues, n.Description)
 		})
 	}
+	os.Exit(parseErrorCount)
```

Comment:
This is unsafe, what if there's 256 parse errors?

Reply:
Ahh, good point. I'll fix.